fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryPredicateFilterPlan.java

```diff
@@ -167,8 +167,9 @@ public int hashCodeWithoutChildren() {
     }
 
     @Override
-    public int planHash() {
-        return getInnerPlan().planHash() + getPredicate().planHash();
+    public int planHash(PlanHashKind hashKind) {
+        // TODO: BaseSource?
+        return getInnerPlan().planHash(hashKind) + getPredicate().planHash(hashKind);
```

Comment:
This is not actually active code and will be changed soon anyways. But for now it would be just consistent to add the source.

Reply:
Issue here is: Source/BlankSource/RecordTypeSource/RepeatedFieldSource do not have hashCode implemented - use Object.hashCode, which is not stable for our use...